apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/CloudTrace.cs

```diff
@@ -119,13 +120,20 @@ private CloudTrace(string projectId, TraceConfiguration config = null, TraceServ
         /// <summary>
         /// Initialize tracing for this application.
         /// </summary>
-        /// <param name="projectId">The Google Cloud Platform project ID.</param>
         /// <param name="application">The Http application.</param>
+        /// <param name="projectId">Optional if running on Google App Engine or Google Compute Engine.
+        ///     The Google Cloud Platform project ID. If running on GAE or GCE the project ID will be
```

Comment:
This last sentence seems to imply that if running on GAE or GCE, any specified project ID will be ignored because it will be detected from the platform, but this does not seem to be the case. I would change this to something like "If **unspecified and** running on..."

There are a few more instances of this below.